src/App.js

```diff
@@ -10,13 +10,6 @@ import AddItem from './components/AddItem';
 import ItemList from './components/ItemList';
 import Home from './pages/Home';
 
-import {
-  getToken,
-  words,
-  calculateEstimate,
-} from '@the-collab-lab/shopping-list-utils';
-import { useNavigate } from 'react-router-dom';
-
 function App() {
```

Comment:
with this removed here, you probably could have resolved the issue #29 along with this feature. I think that would be fine.

Reply:
We were on the fence as to combining issues, even though #29 was so straight forward. Would love to hear advice when this is an acceptable practice, or maybe it's just a case by case basis?